webpack/assets/javascripts/react_app/components/componentRegistry.js

```diff
@@ -58,16 +58,16 @@ const componentRegistry = {
 };
 
 const coreComponets = [
-  { name: 'BookmarkContainer', type: BookmarkContainer },
+  { name: 'BreadcrumbBar', type: BreadcrumbBar },
   { name: 'DonutChart', type: DonutChart },
-  { name: 'StatisticsChartsList', type: StatisticsChartsList },
-  { name: 'PowerStatus', type: PowerStatus },
+  { name: 'FactChart', type: FactChart },
   { name: 'NotificationContainer', type: NotificationContainer },
-  { name: 'ToastNotifications', type: ToastsList, data: false },
-  { name: 'StorageContainer', type: StorageContainer },
+  { name: 'PowerStatus', type: PowerStatus },
   { name: 'PasswordStrength', type: PasswordStrength },
-  { name: 'BreadcrumbBar', type: BreadcrumbBar },
```

Comment:
I am puzzled. Why did you change the order of the `coreComponents` alphabetically?